src/wp-includes/fonts/class-wp-font-library.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Font Library class.
+ *
+ * This file contains the Font Library class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.4.0
+ */
+
+/**
+ * Font Library class.
+ *
+ * @since 6.4.0
+ */
+class WP_Font_Library {
+
+	/**
+	 * Provide the expected mime-type value for font files per-PHP release. Due to differences in the values returned these values differ between PHP versions.
+	 *
+	 * This is necessary until a collection of valid mime-types per-file extension can be provided to 'upload_mimes' filter.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $php_version_id The version of PHP to provide mime types for. The default is the current PHP version.
+	 *
+	 * @return Array A collection of mime types keyed by file extension.
+	 */
+	public static function get_expected_font_mime_types_per_php_version( $php_version_id = PHP_VERSION_ID ) {
+
+		$php_7_ttf_mime_type = $php_version_id >= 70300 ? 'application/font-sfnt' : 'application/x-font-ttf';
```

Comment:
```suggestion
		$php_7_ttf_mime_type = $php_version_id >= 70300 ? 'application/font-sfnt' : 'application/x-font-ttf';
```
Just removing an unnecessary line break. 🙂